stock_available_to_promise_release/models/stock_move.py

```diff
@@ -403,3 +409,10 @@ def _release_split(self, remaining_qty):
         new_move._assign_picking()
 
         return new_move.with_context(context)
+
+    def _assign_picking_post_process(self, new=False):
+        super()._assign_picking_post_process(new)
+        highest_priorities = self.mapped("move_dest_ids.picking_id.priority")
+        if highest_priorities:
+            highest_priorities.sort()
+            self.picking_id.write({"priority": highest_priorities[-1]})
```

Comment:
```suggestion
            self.picking_id.write({"priority": max(highest_priorities)})
```
I suppose… If you agree, you can apply the same everywhere else.

Reply:
```python
>>> max(["1", "12", "42", "0"])
'42'
```

Reply:
Thanks, I updated the pr